src/software/jetson_nano/ansible/playbooks/remote_flash.yml

```diff
@@ -49,17 +41,35 @@
           when: "'stop' in actions"
           tags: always
 
+        - name: Delete everything on ~/
+          file:
+            state: absent
+            path: /home/robot/thunderbots_binaries
+          become_method: sudo
+          become: true
+          register: result
+
         - name: Sync Binaries
+          with_items:
+            # bazel makes its own zip files, we append another zip to deconflict
+            - ../../display/robot_ui_zip.zip
+            - ../../broadcasts/robot_broadcast_sender_zip.zip
+            - ../../thunderloop_main
+          loop_control:
+            loop_var: binary_path
           ansible.posix.synchronize:
-            src: "{{ binary_paths[service_name] }}"
+            src: "{{ binary_path }}"
```

Comment:
this will break our ability to tag/select which services to sync

Reply:
put it back, can't remember why I removed it